specs/PermissionManagement.md

```diff
@@ -0,0 +1,554 @@
+Permission Management APIs
+===
+
+# Background
+
+The WebView2 team is adding support for more permission management scenarios. With the
+extended `PermissionRequested` event args and new permission APIs you will be able to:
+- Set permission for a site at any time
+- List all nondefault permissions
+- Control more permission kinds
+
+The new `ShouldPersist` property on the `PermissionRequestedEventArgs` lets you
```

Comment:
`ShouldCache` looks like a more descriptive name. But better would be a name such that this can default to false.